core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -225,8 +232,23 @@ public Response getBucket(@HeaderParam("Authorization") String authorization,
           .setContinuationToken(continuationToken)
           .setStartAfter(startAfter);
 
-      String path = S3RestUtils.parsePath(AlluxioURI.SEPARATOR + bucket);
+      String path = S3RestUtils.parsePath(String.format("%s%s", AlluxioURI.SEPARATOR, bucket));
       final FileSystem fs = getFileSystem(authorization);
+
+      if (tagging != null) { // GetBucketTagging
```

Comment:
This is checking the for the presence of the query parameter `?tagging` in the URI. Since this key takes no value, it will be an empty string if present, and `null` otherwise.